Runtime/Haptics/HapticPulser.cs

```diff
@@ -1,10 +1,9 @@
 ﻿namespace Zinnia.Haptics
 {
-    using Malimbe.PropertySerializationAttribute;
-    /*using Malimbe.PropertySetterMethod;*/
-    /*using Malimbe.PropertyValidationMethod;*/
-    using Malimbe.XmlDocumentationAttribute;
     using UnityEngine;
+    using Malimbe.XmlDocumentationAttribute;
```

Comment:
The order of these three imports isn't correct. Let's order in alphabetical order.